massa-api/src/public.rs

```diff
@@ -186,6 +187,8 @@ impl MassaRpcServer for API<Public> {
                     operation_datastore: op_datastore,
                 }],
                 is_final,
+                coins: None,
```

Comment:
I think having `coins` here is redundant: you could only have it in the ExecutionStackElement above